bundles/AdminBundle/public/js/pimcore/document/tree.js

```diff
@@ -300,27 +300,7 @@ pimcore.document.tree = Class.create({
             index: index
         };
 
-        if(
-            newParent.data.id !== oldParent.data.id &&
-            (node.data.type === 'page' || node.data.type === 'hardlink') &&
-            pimcore.globalmanager.get("user").isAllowed('redirects')
-        ) {
-            this.nodesToMove.push({
```

Comment:
should we also get rid of nodesToMove property?

Reply:
Removed.